packages/launch/src/focused-launch/plan-details/index.tsx

```diff
@@ -3,22 +3,80 @@
 /**
  * External dependencies
  */
-import React from 'react';
-import { Link } from 'react-router-dom';
+import * as React from 'react';
+import { useSelect, useDispatch } from '@wordpress/data';
 import { __ } from '@wordpress/i18n';
+import { Plans } from '@automattic/data-stores';
+import PlansGrid from '@automattic/plans-grid';
+import { Title, SubTitle } from '@automattic/onboarding';
+import { useHistory } from 'react-router-dom';
 
 /**
  * Internal dependencies
  */
-import { Route } from '../route';
+import { LAUNCH_STORE } from '../../stores';
+import GoBackButton from '../go-back-button';
```

Comment:
Can we use Back Button from `@automattic/onboarding`? Seen [here](https://github.com/Automattic/wp-calypso/pull/47404/files#diff-e51e01d27d081dc244f31143d0841e80254d49c01cee0cb7135bb8df8fa22a62R8)

Reply:
GoBackButton should use BackButton maybe and then reuse here and in DomainDetails View. See also https://github.com/Automattic/wp-calypso/issues/47497#issuecomment-729567924

Reply:
Great idea. Didn't know about this one.